NetKAN/ABCORS.netkan

```diff
@@ -0,0 +1,16 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "ABCORS",
+    "$kref"        : "#/ckan/github/zentarul/abcors",
+    "name"         : "ABookCase Orbital Reference System",
+    "abstract"     : "Hover the mouse on your orbit to display contextual information.",
+    "license"      : "MIT",
+    "$vref"        : "#/ckan/ksp-avc",
+    "resources": {
+        "homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/131706-1",
+        "github": "https://github.com/zentarul/abcors"
```

Comment:
I used the gui, so I'm not sure what you are talking about, I'm in the IRC right now though if you could give me 5 minutes to help me.